### ISSUE TITLE
MissingDependencyError

### ISSUE THREAD

**Issue by tizmagik** [pastelsky/bundlephobia#39]
## Type

Hello! for [react-tracking@5.2.0](https://bundlephobia.com/result?p=react-tracking@5.2.0) bundlephobia correctly threw `MissingDependencyError` (thanks for that! 🙏 )

I [thought I fixed it](https://github.com/NYTimes/react-tracking/commit/63c3e59ac2a5798d9aae60f670c44505549022cc) in [react-tracking@5.2.1](https://bundlephobia.com/result?p=react-tracking@5.2.1) -- but I'm still getting an error 🤔 

## Package name

[react-tracking](https://www.npmjs.com/package/react-tracking)

### Entire error (stringified) I see in my browser console

```
{"error":{"code":"MissingDependencyError","message":"This package (or this version) uses `<code>prop-types</code>`, but does not specify them<br /> either as a dependency or a peer dependency","details":{"name":"MissingDependencyError","originalError":["ModuleNotFoundError: Module not found: Error: Can't resolve 'prop-types' in '/tmp/tmp-build/node_modules/react-tracking/build'","ModuleNotFoundError: Module not found: Error: Can't resolve 'prop-types' in '/tmp/tmp-build/node_modules/react-tracking/build'"],"extra":{"missingModules":["prop-types"]}}}}
```

**Comment by pastelsky** [comment]
Thanks for the report @tizmagik . Bundlephobia has a failure cache — this prevents burdening the server with rebuilding packages that have known to have failed in the past. It appears that the cache key used for this purpose did not account for version changes properly. 

This should be fixed with 6616fd687528aca4bc1b29594e8fab84ebd99eed

**Comment by tizmagik** [comment]
Awesome thank you! Amazing work on this project by the way! 👏
